### DISCUSSION TITLE
Package Wizard for Positron

### DISCUSSION THREAD

**Discussion by anbrav0** [posit-dev/positron#4090] [Feedback]
Currently in RStudio you are able to navigate to Tools > Check for Package Updates and an "Update Packages" wizard nicely pops up with package information, version installed, version available in CRAN and related release news
![image](https://github.com/user-attachments/assets/7ee84b78-9bb7-4288-8039-91cbef3b9334)

However in Positron, I am curious if there is any discussion on if there will be a similar organizational tool that quickly visualizes packages in users system, and if it won't necessarily be similar to the one in RStudios, if there is any hypothetical workflow on how we would be able to quickly visualize something like this 

Thank you all for your wonderful work on this project!

**Comment by juliasilge** [comment]
We have some issues open about managing your packages in various ways (https://github.com/posit-dev/positron/issues/1369 and #1391 come to mind) but I don't believe we yet have a feature request for this particular type of updating wizard. Thanks for bringing it up!

**Reply by anbrav0** [reply]
Thank you for the issue direct @juliasilge !
